core/types/calls.cc

```diff
@@ -631,7 +631,8 @@ DispatchResult dispatchCallSymbol(const GlobalState &gs, const DispatchArgs &arg
                     lines.reserve(alternatives.size());
                     for (auto alternative : alternatives) {
                         auto possibleSymbol = alternative.symbol;
-                        if (!possibleSymbol.isClassOrModule() && !possibleSymbol.isMethod()) {
+                        if (!possibleSymbol.isClassOrModule() &&
+                            (!possibleSymbol.isMethod() || possibleSymbol.data(gs)->isMethodPrivate())) {
```

Comment:
Thanks for the PR! This approach is not quite right. The problem is that this will mean not suggesting private methods when they're actually in scope. Here's a test case:

```ruby
class A
  private def my_private_method; end

  def example
    my_private_metho
  end
end
```

The ticket mentions `isPrivateOk`, which is a property of a method call site (`ast::Send`), not a symbol. You will want to combine the information from the symbol and the send and the symbol.